bitbots_vision/config/visionparams_sim.yaml

```diff
@@ -0,0 +1,97 @@
+    white_color_detector_upper_values_v: 255  # Upper bound for the white color detector value/brightness
+
+    red_color_detector_lower_values_h: 139 # Lower bound for the red color detector hue
+    red_color_detector_lower_values_s: 50 # Lower bound for the red color detector saturation
+    red_color_detector_lower_values_v: 0 # Lower bound for the red color detector value/brightness
+    red_color_detector_upper_values_h: 255 # Upper bound for the red color detector hue
+    red_color_detector_upper_values_s: 255 # Upper bound for the red color detector saturation
+    red_color_detector_upper_values_v: 255 # Upper bound for the red color detector value/brightness
+
+    blue_color_detector_lower_values_h: 90 # Lower bound for the blue color detector hue
+    blue_color_detector_lower_values_s: 197 # Lower bound for the blue color detector saturation
+    blue_color_detector_lower_values_v: 134 # Lower bound for the blue color detector value/brightness
+    blue_color_detector_upper_values_h: 116 # Upper bound for the blue color detector hue
+    blue_color_detector_upper_values_s: 255 # Upper bound for the blue color detector saturation
+    blue_color_detector_upper_values_v: 255 # Upper bound for the blue color detector value/brightness
+
+    field_color_detector_lower_values_h: 30  # Lower bound for the field color detector hue
+    field_color_detector_lower_values_s: 8  # Lower bound for the field color detector saturation
+    field_color_detector_lower_values_v: 58  # Lower bound for the field color detector value/brightness
+    field_color_detector_upper_values_h: 72  # Upper bound for the field color detector hue
+    field_color_detector_upper_values_s: 255  # Upper bound for the field color detector saturation
+    field_color_detector_upper_values_v: 255  # Upper bound for the field color detector value/brightness
+
+    field_boundary_detector_search_method: "downsampling_reversed" # Method for finding the field boundary (iteration, reversed, downsampling_reversed, binary)
+    field_boundary_detector_vertical_steps: 60 # Number of steps on each scanline
+    field_boundary_detector_horizontal_steps: 30 # Number of scanlines
+    field_boundary_detector_roi_height: 4 # Region Of Interest height in which we are looking for green
+    field_boundary_detector_roi_width: 2 # Region Of Interest width in which we are looking for green
+    field_boundary_detector_roi_increase: 0.1 # Value that increases the region of interest if it is located lower in the image
+    field_boundary_detector_green_threshold: 157 # Threshold of green in the area covered by the kernel
+    field_boundary_detector_head_tilt_threshold: 20 # Threshold for the dynamic search method, that describes the head angle at which we are switching between the iteration and the reversed search method.
+
+    line_detector_field_boundary_offset: -15 # Threshold in which we are also searching for lines over the field boundary
```

Comment:
Do you know why we have chosen this value?
Is it to prevent detecting false positive lines, if the field boundary is slightly too high?

Reply:
I think so